src/main/com/yetanalytics/lrs.cljc

```diff
@@ -20,7 +20,9 @@
 (defn get-about
   "Get information about this LRS"
   [lrs auth-identity]
-  (p/-get-about lrs auth-identity))
+  (try (p/-get-about lrs auth-identity)
+       (catch Exception ex
+         {:error ex})))
```

Comment:
Won't you have to update the specs to fit the `or-error` format? (I think you know this already given that you expected the tests to fail.)

Reply:
So that's the funny thing here.. we're not making a change, we're actually fixing a bug. Look a the `:com.yetanalytics.lrs.protocol/error-ret` spec

Reply:
https://github.com/yetanalytics/lrs/pull/71/commits/533a7b411949b0d043d7b6cfad879d21d9dc753e